src/utils/constants.h

```diff
@@ -19,6 +19,17 @@ const byte MENU_ITEM_END_OF_MENU = 8;
 const byte MENU_ITEM_LIST = 9;
 const byte MENU_ITEM_PROGRESS = 10;
 //
+// Control codes
+//
+#define BACKSPACE 8   // Backspace
+#define ENTER     10  // Enter
+#define BACK      27  // Escape
```

Comment:
Does this behave the same way as on a mac?
I noticed that the mac keyboard has different encodings.
I will test this. Just commenting for your knowledge.

Reply:
Should be the same.
The difference is that mac and linux send `\n` (10), and windows sends `\r\n` (13 10).

Reply:
I just checked now and pressing the `ENTER` key prints out. So it's not the same.
`Escape` is detected properly.

```
#CMD# INPUT=13
#CMD# Call=13
```

Reply:
It was unexpected) 
https://stackoverflow.com/questions/38688516/line-break-represented-as-r-on-mac
Looks like need to handle all 3 combinations.

Reply:
Yea, but it's actually okay, cuz it's a completely different key, the key is labeled `return` which matches the code 13 (Carriage Return)

[Apple always going their own direction 😄]